apps/storybook/stories/link/link.stories.tsx

```
@@ -0,0 +1,484 @@
+import { Link } from "@hikoui-beta/react";
```

ESLint

[error] Unable to resolve path to module '@hikoui-beta/react'.

(import-x/no-unresolved)